eddy/plugins/ontology-importer/ontology_importer.py

```diff
@@ -856,6 +856,26 @@ def importPrefixes(self):
                 command = CommandProjectAddPrefix(self.project, prefix, namespace)
                 self.session.undostack.push(command)
 
+    def addImportedOntologies(self):
+        directImports = self.ontology.getDirectImports()
+        importedOnto = set()
+        for imp in directImports:
+            ontologyID = imp.getOntologyID()
+            if ontologyID.isAnonymous():
+                ontologyIRI = None
+                ontologyURI = None
+                ontologyV = None
+            else:
+                ontologyIRI = ontologyID.getOntologyIRI().get().toString()
+                ontologyURI = ontologyID.getOntologyIRI().get().toURI().toString()
```

Comment:
Mentre dal punto di vista della specifica OWL questa operazione è corretta, in pratica non ci basta perché la maggior parte delle volte l'ontologia non è accessibile dal web. Dobbiamo scrivere qui il path al file dell'ontologia, che poi viene propagato nel file graphol.